code/modules/clothing/spacesuits/rig/modules/ds13/kinesis/kinesis.dm

```diff
@@ -505,18 +505,6 @@
 
 	The kinesis module uses fastprocess, ticking 5 times per second
 */
-/obj/item/rig_module/kinesis/proc/start_processing()
```

Comment:
wrapper functions with process in the name should be renamed, but still kept

Reply:
Using proc instead of calling START_PROCESSING is useless here. It's called only in one place. Same applies to stop_processing() proc here